src-tauri/src/cropper/mod.rs

```diff
@@ -107,29 +172,41 @@ fn create_cropper_win(app: &AppHandle) {
             })
             .unwrap();
     }
-
 }
 
 pub fn init_cropper(app: &AppHandle) {
     // Note: we need to create the record button window first
     // Because the JS in cropper window needs a handle to it
-    create_record_button_win(app);
-    create_cropper_win(app);
+    spawn_window(app);
+    // create_record_button_win(app);
+    // create_cropper_win(app);
 }
 
 pub fn toggle_cropper(app: &AppHandle) {
     if !scap::has_permission() {
         crate::open_welcome_window(app);
         return;
     }
-
+    let cursor_position = app.cursor_position();
+    let some_monitor = app.monitor_from_point(1.0, 1.0);
+
+    let current_monitor = monitor_from_point(app).unwrap();
+    let position = PhysicalPosition::new(current_monitor.position().x, current_monitor.position().y);
+    let current_monitor_size = current_monitor.size();
+    let size = PhysicalSize::new(current_monitor_size.width, current_monitor_size.height);
+    
+    // println!("Current monitor ?: {:?}", monitor_from_point(app));
+    println!("CURSOR PHYSICAL POSITION: {:?}", current_monitor.position());
     if let Some(cropper_win) = app.get_webview_window("cropper") {
         match cropper_win.is_visible() {
             Ok(true) => {
                 cropper_win.hide().unwrap();
                 cropper_win
                     .emit("reset-cropper", ())
                     .expect("couldn't reset cropper");
+                    cropper_win.set_position(position);
+                    cropper_win.set_size(size);
+                    println!("Current monitor size on hide: {:?}", cropper_win.inner_size());
                 if let Some(record_button_win) = app.get_webview_window("record") {
                     if record_button_win.is_visible().unwrap() {
                         record_button_win.hide().unwrap();
```

Comment:
Not accounting for the record button here. 
That stuff was refactored afaik.
Will have to rev merge and check up on this.